packages/core/src/common/classes.ts

```diff
@@ -277,10 +277,19 @@ export const OVERLAY_SCROLL_CONTAINER = `${OVERLAY}-scroll-container`;
 export const OVERLAY_START_FOCUS_TRAP = `${OVERLAY}-start-focus-trap`;
 export const OVERLAY_END_FOCUS_TRAP = `${OVERLAY}-end-focus-trap`;
 
-export const PANEL_STACK2 = `${NS}-panel-stack2`;
-export const PANEL_STACK2_HEADER = `${PANEL_STACK2}-header`;
-export const PANEL_STACK2_HEADER_BACK = `${PANEL_STACK2}-header-back`;
-export const PANEL_STACK2_VIEW = `${PANEL_STACK2}-view`;
+export const PANEL_STACK = `${NS}-panel-stack2`;
```

Comment:
Would it make sense to add forwards compatibility to the class names here? For example, we have these classes tied to `.#{$ns}-panel-stack2`. Would it make sense to additionally style to the non-v2 class name as well?
https://github.com/palantir/blueprint/blob/1bf736ac5830d5d01e2f9d7f0140f057f2fe05f7/packages/core/src/components/panel-stack/_panel-stack.scss#L8-L11
```diff
+.#{$ns}-panel-stack,
 .#{$ns}-panel-stack2 {
   overflow: hidden;
   position: relative;
}

Reply:
I like it, let me examine that idea in a follow up.